### ISSUE TITLE
Release 1.1.3 does not seem to work - out/index.js is missing 

### ISSUE THREAD

**Issue by steven-r** [prosegrinder/annotatedtext#304]
While importing this package, I get the error
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/XXX/src/languagetool-cli/node_modules/annotatedtext/out/index.js' imported from /Users/XXX/src/languagetool-cli/dist/annotation-builder.js
    at new NodeError (node:internal/errors:393:5)
    at finalizeResolution (node:internal/modules/esm/resolve:323:11)
    at moduleResolve (node:internal/modules/esm/resolve:916:10)
    at defaultResolve (node:internal/modules/esm/resolve:1124:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

as a workaround I switched back hardcoded to 1.1.2.

**Comment by davidlday** [comment]
My apologies! Thanks for bringing this to my attention. I was simply automating build/deploy. No functional difference between the two versions. Will dig in and get it fixed as soon as I can.

**Comment by davidlday** [comment]
Well, this is embarrassing. When I refactored my shared pipeline, I removed the build step! 😆 Putting it back in and publishing a new version in just a few minutes.

**Comment by davidlday** [comment]
fixed, version 1.1.4 published.
